apps/meeting/CONTRIBUTING.md

```diff
@@ -21,44 +21,7 @@ reported the issue. Please try to include as much information as you can. Detail
 
 
 ## Contributing via Pull Requests
-Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
-
-1. Fork the repository and make sure you are working against the latest source on the *meeting-dev* branch. We use *meeting-dev* branch for `apps/meeting` app to contribute all PRs. This app is used to test the changes releasing into the next [amazon-chime-sdk-component-library-react](https://github.com/aws/amazon-chime-sdk-component-library-react) release.
-2. Run the meeting demo locally first. The *meeting-dev* apps/meeting demo changes are unstable and contributors will need to create and install the `amazon-chime-sdk-component-library-react` tarball from its *master* branch to start the demo.
-
-    1. In a different directory, create `amazon-chime-sdk-component-library-react` tarball.
-    ```
-    git clone https://github.com/aws/amazon-chime-sdk-component-library-react &&
-    cd amazon-chime-sdk-component-library-react &&
-    npm install &&
-    npm run build &&
-    npm pack
-    ```
-    This will create a tarball named: `amazon-chime-sdk-component-library-react-<latest-released-npm-version>.tgz`
-
-    2. Install the tarball in the meeting app
-    ```
-    cd <meeting-dev branch checked out from the aws-samples/amazon-chime-sdk cloned repository>
-    cd apps/meeting
-    npm install
-    npm install <path-to-amazon-chime-sdk-component-library-react-tarball>
-    ```
-
-    3. Start the app:
-    ```
-    npm run start
-    ```
-    
-    4. Open the app using https://0.0.0.0:9000/ in a browser.
-
-3. Modify source and ensure local demo app testing works as expected.
-4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request interface.
-6. The PR should open against the *meeting-dev* branch and not the *main* branch.
-> Note that *meeting-dev* branch is for testing the next release, hence contributor must create PR against the meeting-dev branch.
-The *main* branch is a stable demo branch for the last release and any changes after a release must go into the *meeting-dev* branch.
-
-7. When opening PR, exclude any `package.json` changes for `the amazon-chime-sdk-component-library-react` dependency.
+Contributions via pull requests are much appreciated.
```

Comment:
Do we need legal sign off on this :p

Reply:
Done updated, also removed a stale section in README around contributing.